docs/config_reference.rst

```diff
@@ -322,6 +322,39 @@ System Partition Configuration
    - ``upcrun``: Parallel programs will be launched using the `UPC <https://upc.lbl.gov/>`__ ``upcrun`` command.
    - ``upcxx-run``: Parallel programs will be launched using the `UPC++ <https://bitbucket.org/berkeleylab/upcxx/wiki/Home>`__ ``upcxx-run`` command.
 
+   .. tip::
+
+      ReFrame also allows you to register your own custom launchers simply by defining them in the configuration.
+      Here is an example:
+
```

Comment:
What about expanding this a bit with what users are expected to implement in `command` and move that to a small tutorial of its own? At some we should also have a how-to implement a new backend scheduler.

Reply:
I made some adjustments based on your comment, what do you think now? If we make a longer tutorial about the internals we could have a separate tutorial but I thought it fits nicely in the `tutorial_advanced.html`.